.gitlab/corona-jobs.yml

```diff
@@ -27,11 +27,21 @@ corona_clang_hip:
 # ------------------------------------------------------------------------------
 # HIP + TPLs
 # ------------------------------------------------------------------------------
-corona_clang_hip_tpls:
+corona_rocmcc_tpls:
   parallel:
     matrix:
       - COMPILER_SPEC: rocmcc@5.4.1
         AMDGPU_TARGET: [gfx906]
   variables:
     SPEC: "%${COMPILER_SPEC} cstd=99 cxxstd=14 precision=double ~int64 amdgpu_target=${AMDGPU_TARGET} scheduler=flux +rocm+mpi+magma+raja ^magma+rocm amdgpu_target=${AMDGPU_TARGET} ^raja+rocm~openmp~examples~exercises amdgpu_target=${AMDGPU_TARGET}"
   extends: .corona_build_and_test
+
+corona_rocmcc_tpls_2:
```

Comment:
Is there a reason these two jobs cannot be combined into one?

Reply:
No particular reason, though initially combining the two would result in so many packages installing that the job would timeout. Also, Ginkgo errors with rocm 5.4.1 libraries, so combining the two would require rocm 5.2.3 (or a non 5.4.1 version) to be used

Reply:
Does Ginkgo work with rocm 5.5.0? I see that is now available on Corona.

Reply:
I get an issue while trying to build with 5.5.0. I will test it locally.